src/AddTodoForm.js

```diff
@@ -1,13 +1,23 @@
 import React from "react";
 
-function AddTodoForm (){
+function AddTodoForm (props){
+
+  function handleAddTodo (e){
+    e.preventDefault();
+    let todoTitle = e.target.title.value;
+    e.target.title.value = "";
+    props.onAddTodo(todoTitle);
+    console.log(todoTitle);
```

Comment:
This is good for debugging, but should be removed before merging into what would be a final product.

Reply:
I looked at the lesson and see that they asked you to do this.  Sorry about that.  Nice job following directions 😀

Reply:
Thank you! I totally agree with your comment and will keep that in mind for production ready code.